module_auto_update/addon_hash.py

```diff
@@ -1,5 +1,5 @@
 # Copyright 2018 ACSONE SA/NV.
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
License is LGPL in this case. Why you change it?

Reply:
Replace tricked me :) `http` => `https` it should be